dashboards-observability/public/components/application_analytics/helpers/modal_containers.tsx

```diff
@@ -37,4 +38,34 @@ export const getClearModal = (
       </EuiConfirmModal>
     </EuiOverlayMask>
   );
+};
+
+export const getCustomModal = (
```

Comment:
is it possible to reuse the `getCustomModal` from other places? (i think custom_panels has one)

Reply:
Yes, but I saw that custom panels and notebooks each have their own so I thought it wouldn't hurt to have one for application analytics too. Do you think I should just re-use one of those still?

Reply:
that was probably by accident, notebooks was merged in without changes while custom panels already added this function by that time. But i think it's better to reduce duplicated code

Reply:
On it!